api/preferences/build.gradle.kts

```diff
@@ -0,0 +1,34 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin)
+    alias(libs.plugins.hilt)
+    alias(libs.plugins.ksp)
+}
+
+android {
+    defaultConfig {
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
You don't have tests, save lines and remove this

Reply:
I'll have some fun with gradle in the next PR, promise.